atom/app/atom_main.cc

```diff
@@ -125,7 +132,10 @@ int APIENTRY wWinMain(HINSTANCE instance, HINSTANCE, wchar_t* cmd, int) {
     auto ret = atom::NodeMain(argv.size(), argv.data());
     std::for_each(argv.begin(), argv.end(), free);
     return ret;
-  } else if (IsEnvSet("ELECTRON_INTERNAL_CRASH_SERVICE")) {
+  }
+#endif
+
+  if (IsEnvSet("ELECTRON_INTERNAL_CRASH_SERVICE")) {
```

Comment:
Looks like you've changed the logic here.
This line should be `if (!run_as_node && IsEnvSet("ELECTRON_INTERNAL_CRASH_SERVICE")) {`.

Reply:
This seems OK because there is a `return` directly above